.scalafix.conf

```diff
@@ -1,6 +1,8 @@
 // .scalafix.conf
 rules = [
   RemoveUnused
+//  "github:ohze/scalafix-rules/ConstructorProcedureSyntax"
+//  "github:ohze/scalafix-rules/FinalObject"
```

Comment:
so this was been used locally?

Reply:
Yes. It was been used locally but we could just uncomment the lines to run on CI